scripts/test/__init__.py

```diff
@@ -62,11 +63,11 @@ def test_all(clean_cache: bool = False):
         Step(name="Run Pytest Against Source Code", run=procedures.run_pytest_src),
```

Comment:
Similar to my remark about the `test_dist()` steps, the 3 steps here are the same as in `test_src()`, so can you set things up to only define each set of steps in one place and then use them accordingly?